AUTHORS

```diff
@@ -1396,7 +1396,6 @@ Ville Skyttä                   <scop@cs132170.pp.htv.fi>
 Vincent Pit                    <perl@profvince.com>
 Vishal Bhatia                  <vishal@deja.com>
 Vitali Peil                    <vitali.peil@uni-bielefeld.de>
-vividsnow                      <vividsnow@gmail.com>
 vividsnow                      @vividsnow
```

Comment:
I suspect it's the other one that you want to remove?

Reply:
I assumed vividsnow wanted to hide their email by using the github id, there are a few other cases in AUTHORS like that. I dont have a preference which we use or keep. I just checked the history and the @vividsnow came last, and was part of an automated update which i guess went wrong, so I guess you are right. Ill reverse that change.

Reply:
Reversed. Thanks for the catch @Leont.